### DISCUSSION TITLE
My teammate's account has suspended for no reason.

### DISCUSSION THREAD

**Discussion by VarnostD** [community/community#103291] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello,

My teammate's account has been suspended without any apparent reason, and we are unable to reach anyone for assistance. We have opened a ticket, but there has been no response. The ticket number is 2568173.

It's important to mention that there are vital project-related items inside the account.

Can someone please help us?

Username: yunusemrekosar

**Comment by queenofcorgis** [comment, marked as answer]
Hi @VarnostD,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
